frameworks/Java/officefloor/src/woof_benchmark/src/test/java/net/officefloor/benchmark/TestSuite.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * OfficeFloor - http://www.officefloor.net
+ * Copyright (C) 2005-2018 Daniel Sagenschneider
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package net.officefloor.benchmark;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+/**
+ * Tests.
+ */
+@RunWith(Suite.class)
+@SuiteClasses({ JsonTest.class, DbTest.class, QueriesTest.class, FortunesTest.class, UpdateTest.class,
+		PlaintextTest.class })
+public class TestSuite {
```

Comment:
Thank you for taking the care to make sure the implementation works correctly.  We don't want to include any functional tests in this repository, though.  Could you remove all of these functional tests from the PR?

If you want to maintain those tests on your own outside of this repository, that's fine.  You could add a `.gitignore` to the `frameworks/Java/officefloor` directory with directives to ignore the test files.  And if you want to leave any test-related surefire configuration or stuff like that in the `pom.xml`s (so that you can still run the tests locally without maintaining a separate `pom.xml`) that's also fine.

From our point of view, the TFB toolset's own built-in verification executed by Travis CI are all the functional tests we need.  When frameworks include their own tests, that becomes an additional maintenance burden for us.  We'd rather not maintain any code that isn't essential for the benchmarks.

(The same reasoning applies for startup scripts/configs for running the framework without running the TFB toolset.  We already have a way to run frameworks -- using the toolset -- and we don't want to maintain/support any other ways of running frameworks.  I don't see any files like this in this PR though.)